package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "node-handlebars-boilerplate",
+  "name": "Project 2",
```

Comment:
Tried to run this before merging, and it errored out at this line. Error message stated "Project 2" is an invalid name - a quick search showed that this attribute can only have lowercase letters, hyphens, and underscores.